reana_workflow_controller/rest/workflows_workspace.py

```diff
@@ -255,9 +254,10 @@ def download_file(workflow_id_or_name, file_name):  # noqa
 
         workflow = _get_workflow_with_uuid_or_name(workflow_id_or_name, user_uuid)
 
-        absolute_workflow_workspace_path = os.path.join(
-            current_app.config["SHARED_VOLUME_PATH"], workflow.workspace_path
-        )
+        absolute_workflow_workspace_path = workflow.workspace_path
```

Comment:
I think here you could simply use `workflow.workspace_path` while calling `download_files_recursive_wildcard()`